### ISSUE TITLE
[RA]In certain expansion missions Turkey (Allies house) is used as Soviet house, on the score screen Turkey is counted under Allies casualties

### ISSUE THREAD

**Issue by mvdhout1992** [TheAssemblyArmada/Vanilla-Conquer#295]
That's fixed in the CnCNet patch with modified missions files and a patch to change the country of a house in the map file + score screen casualties count calculation update.

But requires modified mission files so a hard-coded check for those missions would be better I assume. Still should add the modding ability. Hard-coded check should check for sha1.

@tomsons26 says Petroglyph added a hard-coded fix for this which uses scenario name rather than sha1. Need to investigate and update the code to use sha1.

**Comment by hifi** [comment]
Yeah, hardcoding the fix in Vanilla is probably the best way forward. There are already hardcoded things in the original code as well for mission overrides so maybe we can just expand that hack/patch list.
